normalization/src/main/java/zemberek/normalization/TurkishSentenceNormalizer.java

```diff
@@ -152,6 +152,10 @@ public void setAlwaysApplyDeasciifier(boolean alwaysApplyDeasciifier) {
 
   public String normalize(String sentence) {
 
+    if(sentence.matches("^\\P{L}*$")) {
+      return sentence;
```

Comment:
Do you think would sentence.trim.length() == 0 would be easier to understand and faster? Or trim does not catch all cases?

Reply:
I did some tests. Yes, I think it would be more readable and much more faster. I was thinking it might also be useful to eliminate non-letter strings because there's nothing to normalize. But the performance cost is high. It doesn't worth it.